### PR TITLE
Update BN listing and Staff log

### PR DESCRIPTION
**Pull request by hypercyte**
#### osu!standard Probationary BN additions
- [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [Een Persoon](https://osu.ppy.sh/users/10959501)
- [NeKroMan4ik](https://osu.ppy.sh/users/11387664)
- [Karen](https://osu.ppy.sh/users/3143784)
- [Sylas](https://osu.ppy.sh/users/3906405)

### CODE REVIEW

File: wiki/Staff_Log/2020/en.md [hunk 1]
```diff
@@ -325,6 +325,11 @@ Abbreviations for user groups are used throughout this log:
 - 2020-08-10: Added [Murumoo](https://osu.ppy.sh/users/8001433) to **Probationary BN**
 - 2020-08-11: Added [Rocma](https://osu.ppy.sh/users/566276) to **Probationary BN**
 - 2020-08-22: Added [JierYagtama](https://osu.ppy.sh/users/7483452) to **Probationary BN**
+- 2020-08-22: Added [Ryuusei Aika](https://osu.ppy.sh/users/7777875) to **Probationary BN**
```
Comment (cl8n) [inline wiki/Staff_Log/2020/en.md:328]: i think 08-23 for all of these

Comment (hypercyte) [reply wiki/Staff_Log/2020/en.md:328]: oops forgot to change 1sec

**Comment by TPGPL** [comment]
should update translations as well

**Review by TPGPL — APPROVED** [review]
gonna merge this for now as we dont have required language keys anyways

**Pull request merged**